dashboard/src/components/AppView/AppView.tsx

```diff
@@ -198,6 +199,19 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     );
   }
 
+  private renderMigrationNeeded() {
+    return (
+      <div className="banner">
+        <div className="container container-small text-c">
+          <p className="margin-t-small">
+            This release is not being managed by Kubeapps. To be able to upgrade or delete this
+            release <b> click in the button "Migrate" below </b>.
```

Comment:
please use `strong` instead of `b`. Also, there is extra spacing on either side of the text that you should remove.

click in the button "Migrate" below -> click on the "Migrate" button below